app/services/group/v1x1_seed.rb

```diff
@@ -0,0 +1,74 @@
+module Group
+  class V1x1Seed
+    attr_reader :status
+    CATALOG_ADMINISTRATOR_GROUP = "Catalog Administrators".freeze
+
+    def initialize
+      @request = Insights::API::Common::Request.current
+      raise Catalog::NotAuthorized unless @request.user.org_admin?
+    end
+
+    def tenant
+      @request.tenant
+    end
+
+    def code(status)
+      @status = status
+    end
+
+    def process
+      lookup_groups
+      mark_as_seeded
+
+      if @seeded.data.present?
+        code(204)
+      else
+        run_seeding
+        add_user_to_catalog_admin_group
+      end
+
+      self
+    end
+
+    private
+
+    def run_seeding
+      seeded = Insights::API::Common::RBAC::Seed.new(Rails.root.join('data', 'rbac_catalog_seed.yml')).process
```

Comment:
Need to review `rbac_catalog_seed.yml` as it is doing too much not.  We no longer need anything for the "Catalog User" group, users get these permissions by default from the RBAC service and we should not be creating any policy objects.  (Maybe out side of the scope of this PR.)

Reply:
@gmcculloug The Policy keyword is there to bind the roles to the groups. It uses the policy to add roles to_group.
https://github.com/RedHatInsights/insights-api-common-rails/blob/b0305ecd3e1777454add48f05e8a070a02eccdc7/lib/insights/api/common/rbac/seed.rb#L104

Reply:
Got it, not actually trying to create a policy object in RBAC but using the old intermediate object name.  Thanks